### DISCUSSION TITLE
monika not appearing even though i tried using my backup files

### DISCUSSION THREAD

**Discussion by fvrvlan** [Monika-After-Story/MonikaModDev#9502] [general]
help I was just playing MAS when I told Monika I was just gonna restart the game and then when I launch back the game she's gone?! I deleted the MAS file in the Renpy folder and put my backup file and she's still gone?! when I launched a new game all of my progress are GONE. please help Idk what to do now and I don't want to restart everything. oh and i don't even know why the hell in the persistent files the years were 2023/2024. thanks! 

![Screenshot (3610)](https://user-images.githubusercontent.com/113247232/189486659-da205a64-3a86-4dae-bc73-1ba9373f7121.png)
![Screenshot (3609)](https://user-images.githubusercontent.com/113247232/189486600-970b7705-22e4-461e-9b7a-3c8674181a3c.png)
![screenshot0022](https://user-images.githubusercontent.com/113247232/189486734-cf35a0f5-997e-439c-8c15-44f157699669.png)

**Comment by Booplicate** [comment]
Send your aff_log, please.

**Reply by Booplicate** [reply]
Please, send the file itself

**Reply by fvrvlan** [reply]
oh alright, hold on

**Reply by fvrvlan** [reply]
[aff_log.log](https://github.com/Monika-After-Story/MonikaModDev/files/9552273/aff_log.log)

**Reply by Booplicate** [reply]
Send all your persistents (can put them in  a zip)
